### PR TITLE
Style: Improve upon UI/UX of DeepDive

### PR DESCRIPTION
**Pull request by pastelsky**
**Description of PR**
Improves upon the basic UI/ UX of the deep dive page:

- Use sample size instead of the growing "Awaiting data" count.
- Use subtler colors in charts
- Optimize charts for mobile – packed tighter and less wasted white-space.

**Type of PR**
- [x] Enhancement

**Relevant Issues**  

- [x] Compiles and passes lint tests
- [x] Properly formatted
- [x] Tested on desktop
- [x] Tested on phone

### CODE REVIEW

**Comment by vedadeepta** [comment]
Facing the same issue with my PR. i think before `npm ci`, `npm install` command is needed.

File: src/components/deepdive.js [hunk 1]
```diff
@@ -43,36 +43,33 @@ function DeepDive(props) {
...
           <AllStatesChart
-            title="States - Total Cases"
+            title="Total Cases by State"
             data={statesTimeSeries}
           />
         </div>
 
         <div className="card fadeInUp" style={{animationDelay: '0.7s'}}>
-          <GenderChart title="Patient Gender" data={rawData} />
+          <GenderChart title="Patient by Gender" data={rawData} />
```
Comment (s-naveen) [inline src/components/deepdive.js]: Should be Patients by Gender

File: src/components/Charts/chart-defaults.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    position: 'average',
+    displayColors: false,
+    borderWidth: 0,
+    titleSpacing: 4,
+    titleMarginBottom: 8,
+    bodySpacing: 8,
+    cornerRadius: 3,
+    xPadding: 10,
+    yPadding: 8,
+    caretSize: 0,
```
Comment (s-naveen) [inline src/components/Charts/chart-defaults.js]: I feel tooltip is too big now and some part hides behind bottom of the chart. Need to revisit this.

File: src/components/Charts/allstates.js [hunk 3]
```diff
@@ -94,7 +70,7 @@ function AllStatesChart(props) {
     }
 
     datasets.push({
-      borderWidth: 1.5,
+      borderWidth: 3,
```
Comment (s-naveen) [inline src/components/Charts/allstates.js]: it looks good to me but some felt thin lines are better. @sudevschiz @shuklaayush @JunaidBabu thoughts?

Comment (pastelsky) [reply src/components/Charts/allstates.js]: I found the updated colour palette for these lines to not work very well with very thin lines. But if you think the thinner ones looked better, I've updated it to `2` now.

File: src/components/deepdive.js (hunk 1 above)
Comment (pastelsky) [reply src/components/deepdive.js]: Good catch, fixed

File: src/components/Charts/chart-defaults.js (hunk 2 above)
Comment (pastelsky) [reply src/components/Charts/chart-defaults.js]: While the current tooltip is concise, I did find issues when trying to actually read the contents with small text on a semi-transparent background.

image
image



I've updated this to use tighter spacing now (still a tad little roomier than exists today)

**Comment by pastelsky** [comment]
@s-naveen Could take another look at this?

**Review by jeremyphilemon — APPROVED** [review]
LGTM, thanks for the PR! :D

**Pull request merged**